server/index.js

```diff
@@ -68,7 +67,40 @@ app.get('/comments/:id', async (req, res) => {
   } catch (error) {
     console.error(error);
     return res.status(400).json({
-      succes: false,
+      success: false,
+      msg: error,
+    });
+  }
+});
+
+app.get('/user/comment/:id', async (req, res) => {
+  try {
+    const { id } = req.params;
+
+    const comment = await db.getUserComment(id);
+
+    if (!comment || id > 100) {
+      return res.status(400).json({
+        success: false,
+        msg: `no comment with id ${id}`,
+      });
+    }
+
+    if (comment.length === 0) {
+      return res.status(400).json({
+        success: false,
+        msg: `comment ${id} doesn't have comments`,
+      });
+    }
+
+    return res.status(200).send({
+      success: true,
+      data: comment,
+    });
+  } catch (error) {
+    console.error(error);
+    return res.status(400).json({
```

Comment:
Does this `catch` block fire only upon bad requests, or would it capture `500` errors too? (Same comment throughout the `catch` blocks.)

Reply:
These catch blocks were inherited as well.  I have explicitly handled bad request, these could represent just 500. However, I don't want touch these yet. These are sent to front end, hence want to leave them as is for now.